awx/main/models/workflow.py

```diff
@@ -184,6 +184,12 @@ class WorkflowJobNode(WorkflowNodeBase):
         default={},
         editable=False,
     )
+    do_not_run = models.BooleanField(
+        default=False,
+        help_text=_("Indidcates that a job will not be created when True. Workflow runtime "
+                    "semantics will mark this True if the node is in a path that will "
+                    "decidedly not be ran. A value of False means the node may not run."),
```

Comment:
I hate to be making suggestions at this point but

> A value of False means the node may not run.

"A value of False means this node may or may not run."?

Perhaps it would be best to delete that sentence. I don't think we need it.